test/functional/apps/visualize/_area_chart.js

```diff
@@ -0,0 +1,216 @@
+define(function (require) {
+  var Common = require('../../../support/pages/Common');
+  var HeaderPage = require('../../../support/pages/HeaderPage');
+  var SettingsPage = require('../../../support/pages/settings_page');
+  var DiscoverPage = require('../../../support/pages/DiscoverPage');
+  var VisualizePage = require('../../../support/pages/VisualizePage');
```

Comment:
Thanks.  I like that.  Maybe I'll incorporate that along with snake_casing in another PR.